src/pages/NewPassword/NewPassword.tsx

```diff
@@ -0,0 +1,28 @@
+import { useNavigate } from 'react-router-dom';
+import Button from '../../components/Button/Button';
+import Form from '../../components/Form/Form';
+import FormInputs from '../../components/FormInputs/FormInputs';
+import Input from '../../components/Input/Input';
+import Popup from '../../components/Popups/Popup/Popup';
+import { useTranslation } from 'react-i18next';
+
+const NewPassword = () => {
+    const { t } = useTranslation();
+    const navigate = useNavigate();
+    const onSubmit = () => {
+        navigate('/password_done');
+    };
+    return (
+        <Popup title={t('pages.newPassword.title')}>
+            <Form name="form-password-new" onSubmit={onSubmit}>
+                <FormInputs>
+                    <Input type="text" name="input_newPassword" placeholder="******" nameLabel={t('pages.newPassword.nameLabel')}></Input>
```

Comment:
```suggestion
                    <Input type="text" name="newPassword" placeholder="******" nameLabel={t('pages.newPassword.nameLabel')}></Input>
```
Слово input здесь лишнее так как не дает новой информации 
и в остальных формах тоже. Можешь поменять в этом ПРе, а можешь в соответственные тикеты дописать "убрать префикс input_ из атрибута name у инпутов"

Reply:
Уберу

Reply:
Но это не моих рук дело это я с мэйна запуливал